CMSgov/mint-app
# [EASI-2191] Clean up array helpers and radio button components

# EASI-2191

## Changes and Description

- Removed `arrayHelper` throughout the codebase
- Removed `onChange` part of the  Radio buttons, since it is not needed



## How to test this change

1. Go through the app and check to see all data persists, is in read only, and in CSV
2. Try to break the app



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
